src/modules/lite/explorer/components/VoteDetails.tsx

```diff
@@ -174,17 +187,17 @@ export const VoteDetails: React.FC<{ poll: Poll | undefined; choices: Choice[];
             justifyContent={isMobileSmall ? "flex-start" : "flex-end"}
           >
             <Typography color="textPrimary" variant="body1">
-              {nFormatter(calculateProposalTotal(choices, tokenData?.decimals), 1)}
+              {numbro(calculateProposalTotal(choices, isXTZ ? 0 : tokenData?.decimals)).format(formatConfig)}
```

Comment:
XTZ decimals is 18 by default. Unless it's a different case with the data we're getting in this case from the API or Homebase Backend.

Reply:
fix!